sdk/core/Azure.Core/tests/public/PatchModels/ChildPatchModel.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core.Json;
+
+namespace Azure.Core.Tests.PatchModels
+{
+    /// <summary>
+    /// This model illustrates a nested child model in a parent model.
+    /// </summary>
+    public partial class ChildPatchModel
+    {
+#pragma warning disable AZC0020 // Avoid using banned types in libraries
+        private readonly MutableJsonElement _element;
```

Comment:
I'll see if I can add some benchmarks to see how favorable that approach is.

Reply:
Per https://github.com/annelo-msft/azure-sdk-for-net/pull/9, MJD serialization is comparable to standard model serialization, except when we modify the change list, since it's essentially a pass-through to JsonDocument when there are no changes.  I'll spend some time looking at whether we can optimize the case where we access the change list.